src/Hedgehog/CSharp/Property.fs

```diff
@@ -17,14 +17,17 @@ type Property private () =
         Property.discard
 
     static member Success (value : 'T) : Property<'T> =
-        Property.Success(value)
+        Property.success value
 
     static member FromBool (value : bool) : Property<unit> =
         Property.ofBool(value)
 
     static member FromGen (gen : Gen<Journal * Result<'T>>) : Property<'T> =
         Property.ofGen gen
 
+    static member FromResult (result : Result<'T>) : Property<'T> =
+        Property.ofResult result
+
```

Comment:
Yeah, if we could generate all src/Hedgehog/CSharp/*.fs files via T4 it'd be awesome (T4 is a pain to work with OTOH). :+1: